public/assets/bootstrap/js/bootstable.js

```diff
@@ -236,7 +237,11 @@ function butRowEdit(but) {
                 input = '<select class="form-control input-sm" >';
 
                 opts.forEach(function (op) {
-                    input += '<option>' + op + '</option>';
+                    if(op == $category) {
```

Comment:
Change $category to cont.
`                    if(op == cont) {`

Reply:
Done